gpflow/kernels/multioutput/kernels.py

```diff
@@ -73,7 +73,9 @@ def K_diag(self, X, full_output_cov=True):
         """
         raise NotImplementedError
 
-    def __call__(self, X, X2=None, *, full_cov=False, full_output_cov=True):
+    def __call__(self, X, X2=None, *, full_cov=False, full_output_cov=True, presliced=False):
```

Comment:
The change from the `full_cov=True` default in the `Kernel` base class to `full_cov=False` here is another reason that makes me think it shouldn't inherit from `Kernel` at all... (out of scope of this PR of course)